lookout/core/cmdline.py

```diff
@@ -64,7 +64,7 @@ def run_analyzers(args):
     :param args: Parsed command line arguments.
     :return: None
     """
-    slogging.setup(args.log_level, args.log_structured)
+    slogging.setup(args.log_config_path, args.log_level, args.log_structured)
```

Comment:
I want to make the configuration file optional - so better to put it to the last place with an empty default value.